app/views/certificates/status/_certificate_table.html.erb

```diff
@@ -2,36 +2,55 @@
   <caption class="govuk-table__caption govuk-heading-m"><%= t('certificates.caption', heading: heading) %></caption>
   <thead class="govuk-table__head">
   <tr class="govuk-table__row">
-    <th class="govuk-table__header"><%= t 'certificates.header_last_updated' %></th>
-    <th class="govuk-table__header"><%= t 'certificates.header_id' %></th>
-    <th class="govuk-table__header"><%= t 'certificates.header_usage' %></th>
-    <th class="govuk-table__header"><%= t 'certificates.header_enabled' %></th>
-    <th class="govuk-table__header" scope="col"></th>
+    
+    <th class="govuk-table__header"><%= t 'certificates.header.details' %></th>
+    <th class="govuk-table__header"><%= t 'certificates.header.usage' %></th>
+    <th class="govuk-table__header"><%= t 'certificates.header.enabled' %></th>
+    <th class="govuk-table__header"><%= t 'certificates.header.last_updated' %></th>
+    <th class="govuk-table__header"><%= t 'certificates.header.action' %></th>
   </tr>
   </thead>
   <tbody class="govuk-table__body">
    <% certificates.each do |certificate| %>
       <tr class="govuk-table__row" id="certificate_table_<%= certificate.id %>">
-        <th class="govuk-table__header" scope="row"><%= format_date_time(certificate.updated_at) %></th>
-        <td class="govuk-table__cell"><%= certificate.id %></td>
+        <td class="govuk-table__cell">
+          ID: <%= certificate.id %> <br>
+          Subject: <strong><%= certificate.x509.subject %></strong> <br>
+          Expires: <%= certificate.x509.not_after %> <br>
+          Value:
```

Comment:
I know this is admin so doesn't particularly matter, but for consistency with the rest of the file should these be localised

Reply:
yep can do

Reply:
updated